makefile

```
@@ -1,4 +1,5 @@
 VERSION ?= $(shell git describe --tags)
+APP_VERSION ?= $(shell git describe --abbrev=0)
```

```suggestion
APP_VERSION ?= $(shell git describe --tags --abbrev=0)
```

In case the tag annotation is missing